dom/src/test/java/org/apache/james/mime4j/field/ContentDispositionFieldTest.java

```diff
@@ -64,7 +64,7 @@ public void testGetParameter() throws Exception {
 
```

Comment:
How about LenientContentDispositionFieldTest ? 

We should avoid divergences between both of them...